pom.xml

```diff
@@ -35,54 +41,7 @@
 		<warContext>JDBCApp</warContext>
 		<package.file>${project.build.directory}/${project.artifactId}.zip</package.file>
 		<packaging.type>usr</packaging.type>
-</properties>
-
-
-	<profiles>
-		<profile>
-			<id>runnable</id>
-			<properties>
-				<package.file>${project.build.directory}/${project.artifactId}.jar</package.file>
-				<packaging.type>runnable</packaging.type>
-			</properties>
-		</profile>
-		<profile>
-			<id>liberty-test</id>
-
-			<!--Do not start the server unless testing -->
-
-			<activation>
-				<property>
-					<name>!skipTests</name>
-				</property>
-			</activation>
-
-			<build>
-				<plugins>
-					<plugin>
-						<groupId>net.wasdev.wlp.maven.plugins</groupId>
-						<artifactId>liberty-maven-plugin</artifactId>
-						<executions>
-							<execution>
-								<id>start-server</id>
-								<phase>pre-integration-test</phase>
-								<goals>
-									<goal>start-server</goal>
-								</goals>
-							</execution>
-							<execution>
-								<id>stop-server</id>
-								<phase>post-integration-test</phase>
-								<goals>
-									<goal>stop-server</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
+	</properties>
```

Comment:
I added the profile back in but couldn't they use the include runnable option to achieve something similar (https://github.com/WASdev/ci.maven/blob/master/docs/package-server.md)